styles/variables-gallium.less

```diff
@@ -108,7 +108,7 @@
 
 // PopupMenu
 // ---------------------------------------
-@agate-popupmenu-title-margin-bottom: 136px;
+@agate-popupmenu-title-margin: 136px;
```

Comment:
To match with the previous version of the implementation, shouldn't this be `0 0 136px 0`?